### PR TITLE
actually use the check kwarg

### PR DESCRIPTION
**Pull request by CJ-Wright**
Checklist
* [ ] Added a ``news`` entry

### CODE REVIEW

File: conda_smithy/cli.py [hunk 1]
```diff
@@ -342,14 +342,15 @@ def __init__(self, parser):
             help="Exclusive conda-build config file to replace conda-forge-pinning. "
             + "For advanced usage only",
         )
-        scp.add_argument("--check", action="store_true", default=False, help="Check if regenerate can be performed")
+        scp.add_argument("--check", action="store_true", help="Check if regenerate can be performed")
```
Comment (scopatz) [inline conda_smithy/cli.py]: I think you should still have `default=False`

**Comment by scopatz** [comment]
This good now?

**Comment by CJ-Wright** [comment]
I think so!

**Comment by CJ-Wright** [comment]
Thank you very much!

**Comment by CJ-Wright** [comment]
I'll do that rn

**Pull request merged**